Makefile

```diff
@@ -1,7 +1,7 @@
 REG := 100225593120.dkr.ecr.us-east-1.amazonaws.com
-DOCKER_PULL_TAG := latest
-DOCKER_BUILD_TAG := latest
-ALLIANCE_RELEASE := 5.0.0
+DOCKER_PULL_TAG := stage
+DOCKER_BUILD_TAG := stage
+ALLIANCE_RELEASE := 6.0.0
```

Comment:
Can you change the Makefile to keep latests as default but define stage when calling the Makefile for stage?

In the Makefile you would need
```
DOCKER_PULL_TAG ?= latest
DOCKER_BUILD_TAG ?= latest
ALLIANCE_RELEASE ?= 6.0.0
```
And then when calling you would do something like
```bash
make <stage_target_name> DOCKER_PULL_TAG=stage DOCKER_BUILD_TAG=stage
```

I don't think we want to change the defaults to stage and we definitely wouldn't want stage as a default to propagate to other branches.

Reply:
@mluypaert Didn't @oblodgett remove the latest tags from AWS for Docker images? That was the reason for the change. I'll check real quick.

Reply:
Looks like "latest" is still there for this repo, but it's been removed for most others. Do we want to make an exception for this file repo and keep stage?

Reply:
@christabone Perhaps make it the same as used in the loader:
```
DOCKER_PULL_TAG?=stage
DOCKER_BUILD_TAG?=latest
```

That way the default protects us from accidental overwrites and anyone using the Makefile to build images needs to be intentional if they want to build images for stage (or any other environment).

Reply:
That sounds great @mluypaert. Thanks a bunch for the input and the edits.

Reply:
@mluypaert Does that change look correct?

Reply:
@mluypaert following the same logic ALLIANCE_RELEASE should be set to 0.0.0 so that it also would need to be specified. But yes I would partially agree that the LATEST tag is no longer and the make will fail with "latest" I would prefer that as it would for the person to choose the tags and release version.

Reply:
> @mluypaert Does that change look correct?

Yup, fine by me now.

Reply:
@oblodgett Updated the release to 0.0.0 as per your suggestion.